src/sse.ts

```diff
@@ -1,8 +1,7 @@
 import chalk from 'chalk'
-import { compose, forEach, contains, path, pathOr } from 'ramda'
-
```

Comment:
nit: Not sure what coding standard you're following in this repo, but usually we leave an empty line between package imports and local imports.

Reply:
We'll shortly configure the linter to do this automatically